docs/components/power-sensor/_index.md

```diff
@@ -297,6 +297,59 @@ readings, err := myPowerSensor.Readings(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
+### DoCommand
+
+Execute model-specific commands that are not otherwise defined by the component API.
+If you are implementing your own power sensor and add features that have no built-in API method, you can access them with `DoCommand`.
+
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+**Parameters:**
+
+- `command` [(Dict[str, Any])](https://docs.python.org/3/library/stdtypes.html#typesmapping): The command to execute.
+
+**Returns:**
+
+- [(Dict[str, Any])](https://docs.python.org/3/library/stdtypes.html#typesmapping): Result of the executed command.
+
+```python {class="line-numbers linkable-line-numbers"}
+my_sensor = PowerSensor.from_robot(robot=robot, name="my_power_sensor")
+
+reset_dict = {
+  "command": "reset",
+  "example_param": 30
+}
+
+do_response = await my_sensor.do_command(reset_dict)
+```
+
+For more information, see the [Python SDK Docs](https://python.viam.dev/autoapi/viam/components/power_sensor/client/index.html#viam.components.power_sensor.client.PowerSensorClient.do_command).
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+
+**Parameters:**
+
+- `ctx` [(Context)](https://pkg.go.dev/context): A Context carries a deadline, a cancellation signal, and other values across API boundaries.
+- `cmd` [(map[string]interface{})](https://go.dev/blog/maps): The command to execute.
```

Comment:
Question: I'm vbad at go, but should this be `cmd` --> `command` to match usage in the Go example below? If so, I see this elsewhere in the docs (as `cmd`) ([SLAM](https://docs.viam.com/services/slam/#docommand) for example) as well.

Reply:
It is named cmd, but the example demonstrates feeding it a mapping with an example command named command! See https://pkg.go.dev/go.viam.com/rdk@v0.12.0/resource#Resource (which should really be linked to here, updating). Is that still confusing?

Reply:
No that makes sense! I think the updated link helps clarify also. Thanks much!!